doc/api/test.md

```diff
@@ -1611,10 +1611,11 @@ and `clearInterval` functions from [node:timers](./timers.md),
 
 Alternatively, if you call `mock.timers.enable()` without any parameters:
 
-All timers (`'setInterval'`, `'clearInterval'`, `'setTimeout'`, and `'clearTimeout'`)
-will be mocked. The `setInterval`, `clearInterval`, `setTimeout`, and `clearTimeout`
-functions from `node:timers`, `node:timers/promises`,
-and `globalThis` will be mocked.
+All timers (`'setInterval'`, `'clearInterval'`, `'setTimeout'`, `'clearTimeout'`,
```

Comment:
Are `clearInterval`, `clearTimeout`, and `clearImmediate` considered timers?
I know this was discussed in a previous PR and I think @ljharb commented, but I can't seem to find it now 😅